packages/image-cropper/ImageCropper.tsx

```diff
@@ -0,0 +1,458 @@
+import clsx from 'clsx';
+import normalizeWheel from 'normalize-wheel';
+import React from 'react';
+
+import {
+  computeCroppedArea,
+  getDistanceBetweenPoints,
+  getMidpoint,
+  restrictPosition,
+  restrictValue
+} from './cropUtils';
+import type { Area, MediaSize, Point, Size } from './types';
+
+export type CropperProps = {
+  image?: string;
+  cropSize: Size;
+  cropPosition: Point;
+  borderSize: number;
+  zoom: number;
+  zoomSpeed: number;
+  onCropChange: (location: Point) => void;
+  onZoomChange?: (zoom: number, maxZoom: number) => void;
+  onCropComplete?: (croppedAreaPixels: Area) => void;
+};
+
+type State = {
+  hasWheelJustStarted: boolean;
+};
+
+type GestureEvent = UIEvent & {
+  scale: number;
+  clientX: number;
+  clientY: number;
+};
+
+class ImageCropper extends React.Component<CropperProps, State> {
```

Comment:
we can just import component instead of importing full react package (tree shaking)

Reply:
In all places 🙇🏼

Reply:
Sure, I'll update. For some types, there seems to be a distinction  between with and without `React.`, they point to different libs, so i have kept it as-is in those places.
For example `MouseEvent` vs `React.MouseEvent`:
`(e: MouseEvent | React.MouseEvent | GestureEvent)`

Reply:
It's fine then, its not a big type issue 🙇🏼

Reply:
We can import them carefully 🙇🏼